WooCommerce/WooCommerceTests/POS/Mocks/MockPOSItemProvider.swift

```diff
@@ -1,11 +1,6 @@
 import Foundation
-import protocol Yosemite.PointOfSaleItemServiceProtocol
-import enum Yosemite.POSItem
-import protocol Yosemite.POSOrderableItem
-@testable import struct Yosemite.POSSimpleProduct
-@testable import struct Yosemite.POSVariation
-import struct Yosemite.PagedItems
-import struct Yosemite.POSVariableParentProduct
```

Comment:
I can put all these back... but the number of imports needed for a mock was just getting silly...